src/js/map/map_download.js

```diff
@@ -1,10 +1,12 @@
-import {Observable, saveAs} from "../utils";
 import html2canvas from 'html2canvas';
 
+import {Observable, saveAs} from "../utils";
+import {MapChip} from "../elements/mapchip";
+
 export class MapDownload extends Observable {
     constructor() {
         super();
-
+        this.mapChip = new MapChip();
```

Comment:
instead of creating a new `MapChip` you can pass in the mapchip from `load.js`

Reply:
@milafrerichs can you elaborate more about this?

Reply:
in `load.js` we have:
```js
const mapchip = new MapChip(config.choropleth);
....
const mapDownload = new MapDownload();
```
you could add the `mapchip` to `mapDownload`

```js
const mapDownload = new MapDownload(mapchip);
```

and then in the constructor:
```js
constructor(mapchip) {
  super();
  this.mapChip = mapchip;
```

Reply:
Done